### DISCUSSION TITLE
Sounds dumb... but my ip address won't change

### DISCUSSION THREAD

**Discussion by Marcbacca** [tteck/Proxmox#3200] [Help Needed]
Hi folks, I installed the nginx proxy manager lxc and set it up etc. I'm  wanting to now decommission the old lxc and swap this one in. However, when I edit the /etc/network/interfaces file to change it's ip addy to match the old server (.25), this isn't being saved upon reboot, just reverts back to the old one (.31) specified upon install. The only command I'm aware of for these promox scripts installs is 'update', which doesn't give an option to change any settings. Can someone please let me know if there is a 'setup' or 'settings' type command I can use to change the static ip of my lxc install? Or a txt file to edit or something.. thanks.

**Comment by tteck** [comment]
Have you tried cloning the MAC address?

**Reply by Marcbacca** [reply]
This is a static address assigned from the original install script. Would cloning the MAC address do anything in this case?

I'm just trying to change the current static ip in the lxc. I'm trying to find the mechanism which is changing the interface file to static upon boot so I can change one number... thanks.
